apis/net/ieee802154/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "libtock_ieee802154"
+version = "0.1.0"
+authors = [
+    "Tock Project Developers <tock-dev@googlegroups.com>",
+    "Wojciech Przytuła <woj.przytula@gmail.com>",
```

Comment:
I'm fine with removing this altogether.

Reply:
Several other crates in the `libtock-rs` repository name an individual in `authors`. I agree that having license headers and documenting attribution there is better, but I'm fine with this as-is (particularly because we don't currently have license headers).